### ISSUE TITLE
Eliminate WORKER_SIDECAR_IMAGE env var?

### ISSUE THREAD

**Issue by chainlink** [pachyderm/pachyderm#4762]
Is there cases where the worker sidecar image would not be pachyderm/pachd at the same version as the pachd master node?

Eliminating `WORKER_SIDECAR_IMAGE` would simplify our manifest, reduce the need to keep the same value in sync across multiple places and help unlock tools like skaffold for local development

**Comment by jdoliner** [comment]
> Is there cases where the worker sidecar image would not be pachyderm/pachd at the same version as the pachd master node?

No, there's no reason to mix versions like this, and it normally causes problems if you do.
